src/main/java/seedu/address/logic/commands/PomCommand.java

```diff
@@ -50,24 +55,46 @@
         STOP;
     }
 
+    /**
+     * PomCommand constructor.
+     *
+     * @param targetIndex of the task to be focused on during pomodoro.
+     */
     public PomCommand(Index targetIndex) {
         this.targetIndex = targetIndex;
         this.timerAmount = -1;
         this.pomType = POM_TYPE.NORMAL;
     }
 
+    /**
+     * PomCommand constructor.
+     *
+     * @param targetIndex of the task to be focused on during pomodoro.
+     * @param timerAmount time in minutes for particular pomodoro cycle.
+     */
     public PomCommand(Index targetIndex, float timerAmount) {
         this.targetIndex = targetIndex;
         this.timerAmount = timerAmount * 60;
         this.pomType = POM_TYPE.NORMAL;
     }
 
+    /**
+     * PomCommand constructor.
```

Comment:
Perhaps it would be good to shed some light on why you would use which constructor in which case?